packages/doctor/lib/general.js

```diff
@@ -5,9 +5,21 @@ import NodeDetector from './node-detector';
 import {util} from '@appium/support';
 import {EOL} from 'os';
 import '@colors/colors';
+import {resolveAppiumHome} from '@appium/support/build/lib/env';
 
 let checks = [];
 
+class AppiumHomeCheck extends DoctorCheck {
+  async diagnose() {
+    return ok(`Default APPIUM_HOME is ${await resolveAppiumHome()}`);
```

Comment:
```suggestion
    return ok(`APPIUM_HOME is ${await resolveAppiumHome()}`);
```

The "default" `APPIUM_HOME` is `~/.appium`, which may not be whatever this is